prow/getting_started_deploy.md

```diff
@@ -308,19 +313,25 @@ gcloud iam service-accounts create prow-gcs-publisher # step 1
 identifier="$(  gcloud iam service-accounts list --filter 'name:prow-gcs-publisher' --format 'value(email)' )"
 gsutil mb gs://prow-artifacts/ # step 2
 gsutil iam ch allUsers:objectViewer gs://prow-artifacts # step 3
-gsutil iam ch "serviceAccount:${identifier}:objectCreator" gs://prow-artifacts # step 4
+gsutil iam ch "serviceAccount:${identifier}:objectAdmin" gs://prow-artifacts # step 4
 gcloud iam service-accounts keys create --iam-account "${identifier}" service-account.json # step 5
 kubectl create secret generic gcs-credentials --from-file=service-account.json # step 6
 ```
 
-Then, set the field in your `config.yaml`:
+Then, setup plank's `default_decoration_config` in `config.yaml`:
 
 ```yaml
 plank:
-  # other config ...
   default_decoration_config:
-    # other config ...
-    gcs_credentials_secret: gcs-credentials
+    utility_images: # it's recommended that these tags match the other prow image tags
+      clonerefs: "gcr.io/k8s-prow/clonerefs:v20190619-25afbb545"
+      initupload: "gcr.io/k8s-prow/initupload:v20190619-25afbb545"
+      entrypoint: "gcr.io/k8s-prow/entrypoint:v20190619-25afbb545"
+      sidecar: "gcr.io/k8s-prow/sidecar:v20190619-25afbb545"
+    gcs_configuration:
+      bucket: prow-artifacts # the bucket we just made
+      path_strategy: explicit
+    gcs_credentials_secret: gcs-credentials # the secret we just made
```

Comment:
Rather than elide over other config, let's add the minimum required fields for `checkconfig` to stop failing